Source/WebKit/Configurations/GPUExtension.xcconfig

```diff
@@ -31,3 +31,6 @@ PRODUCT_BUNDLE_NAME = GPUExtension;
 
 // For simulator builds, entitlements are added to a special __entitlements section on the binary rather than the signature.
 CODE_SIGN_ENTITLEMENTS[sdk=*simulator] = Shared/AuxiliaryProcessExtensions/GPUProcessExtension.entitlements;
+
+OTHER_LDFLAGS_YES = -Wl,-dyld_env,@$(WK_WEBKITADDITIONS_HEADERS_FOLDER_PATH)/dlsym.WebKit.GPU;
```

Comment:
Is this is the right syntax? I expected to see `variable=path`, not just a path.

Reply:
That's a very good point, I'll look closer.

Thanks for reviewing!

Reply:
Is this working? I just don't see how a variable assignment without an assignment can possibly be correct.

Reply:
The @ sign means "read the file and parse its contents as arguments" (compiler tools usually call this a "response file").